src/Tooltip/Tooltip.tsx

```diff
@@ -53,68 +112,162 @@ export class Tooltip extends React.Component<TooltipProps, TooltipState> {
                 case "message-container":
                 case "triangle":
                     return;
-                default: this.setState({ toggle: false });
+                default: this.onTooltipToggle(e, false);
             }
         } else {
-            this.setState({ toggle: false });
+            this.onTooltipToggle(null, false);
         }
     }
 
-    /** Forces the tooltip to show */
-    forceShow() {
-        !this.state.toggle && this.setState({ toggle: true });
-    }
-
-    isPositioned(search: string): boolean {
-        const position: string = this.props.position ? this.props.position : "bottom";
-        return position.search(search) === 0;
-    }
-
-    toggleTooltip(state?: boolean, e?: React.MouseEvent<HTMLDivElement>) {
-        if (state !== undefined) {
-            this.setState({ toggle: state });
-        } else {
-            this.setState({ toggle: !this.state.toggle });
-        }
-
-        this.props.onClick && this.props.onClick(e);
+    /**
+     * Forces the tooltip to show
+     */
+    forceShow = (): void => {
+        !this.state.visible && this.onTooltipToggle(null, true);
     }
 
     render() {
         return (
-            <div className={"tooltip-container" + (this.props.className ? ` ${this.props.className}` : "")} id={this.props.id}>
+            <div
+                className={"tooltip-container" + (this.props.className ? ` ${this.props.className}` : "")}
+                id={this.props.id}
+                ref={this.containerRef}
+            >
                 <div
-                    className="icon tooltip-icon"
-                    onClick={(e: React.MouseEvent<HTMLDivElement>) => !this.props.triggerOnHover && this.toggleTooltip(undefined, e)}
-                    onMouseEnter={() => this.props.triggerOnHover && this.toggleTooltip(true)}
-                    onMouseLeave={() => this.props.triggerOnHover && this.toggleTooltip(false)}
+                    className="tooltip-reference"
+                    aria-describedby={this.state.id}
```

Comment:
Described by what? this should refer to a text element describing it. [See this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)

Reply:
described by the tooltip